setup.py

```diff
@@ -6,7 +6,7 @@
 
 # may be needed in some cases for versioneer to get version correctly during builds
 sys.path.append(os.path.dirname(os.path.abspath(__file__)))
-import versioneer  # noqa: E402
+from setuptools_scm import get_version
```

Comment:
Looks like we need to tell the flake8 linter to ignore that this import isn't at the top of the file with the other imports.

```suggestion
from setuptools_scm import get_version  # noqa: E402
```